generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx.ejs

```diff
@@ -108,7 +108,7 @@ export const UserManagement = (props: RouteComponentProps<any>) => {
       <h2 id="user-management-page-heading" data-cy="userManagementPageHeading">
         <Translate contentKey="userManagement.home.title">Users</Translate>
         <div className="d-flex justify-content-end">
-          <Button className="mr-2" color="info" onClick={handleSyncList} disabled={loading}>
+          <Button className="me-2" color="info" onClick={handleSyncList} disabled={loading}>
```

Comment:
Is 'me' a new class in Bootstrap? I assume margin-end? Should we update other margin classes as well?

Reply:
As far as I know I've updated everything.

Reply:
Most of the changes are on react-jhipster side.